### PR TITLE
T72 modal page

### PR DESCRIPTION
**Pull request by VikashHart**
[Link to Ticket 🎟](https://trello.com/c/mXPRj83z)
[Link to Ticket 🎟](https://trello.com/c/lMP5qehn)
[Link to Ticket 🎟](https://trello.com/c/SwtRjStS)

## Description
[//]: # (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.)
Review page added for reviewing and selecting photos after a shoot.

## Type of change

[//]: # (Please delete options that are not relevant.)

- [x] 🌟New feature (non-breaking change which adds functionality)

## UI Changes 
[//]: # (Add images/gifs if applicable)

simulator screen shot - iphone xr - 2018-12-04 at 18 53 54

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Manually tested
- [x] Unit tests added/updated

## Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Update my ticket with relevant information (inProgress or any comments needed)

### CODE REVIEW

**Comment by VikashHart** [comment]
@vhart

File: PhotoBooth/CameraViewController.swift [hunk 1]
```diff
@@ -92,13 +90,23 @@ class CameraViewController: UIViewController {
                 self?.countdownView.updateWith(timeRemaining: timeRemaining, animated: true)
             } else {
                 self?.countdownView.updateWith(timeRemaining: timeRemaining, animated: false)
+                guard self?.viewModel.capturedImages.count == self?.viewModel.numberOfPhotos else { return }
```
Comment (vhart) [inline PhotoBooth/CameraViewController.swift]: remove

File: PhotoBooth/CameraViewController.swift [hunk 2]
```diff
@@ -92,13 +90,23 @@ class CameraViewController: UIViewController {
...
             }
         }
     }
 
+    private func presentReviewPage(images: [UIImage]) {
+        let reviewVC = ModalReviewViewController(capturedImages: images)
+        present(reviewVC, animated: true, completion: nil)
+        viewModel.viewModelReset()
```
Comment (vhart) [inline PhotoBooth/CameraViewController.swift]: viewModelReset seems like a strange name for the function. Maybe just `reset()`

File: PhotoBooth/CameraViewController.swift [hunk 3]
```diff
@@ -92,13 +90,23 @@ class CameraViewController: UIViewController {
...
+        countdownView.isHidden = true
```
Comment (vhart) [inline PhotoBooth/CameraViewController.swift]: this should be a property of your view model but ok for now

File: PhotoBooth/CountdownTimer.swift [hunk 4]
```diff
@@ -51,6 +51,7 @@ class CountdownTimer: TimerModeling {
     }
 
     @objc private func updateTimer() {
+        NSLog("TIME")
```
Comment (vhart) [inline PhotoBooth/CountdownTimer.swift]: you left a log in

File: PhotoBoothTests/ReviewCellViewModelSpec.swift [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+                    beforeEach {
+                        viewModel.isSelected = true
+                    }
+
+                    it("the selection alpha should be 0") {
+                        expect(viewModel.selectionAlpha).to(be(0.5))
+                    }
+                }
+
+//                context("if selected state changes, the onSelectionChanged callback will be called") {
```
Comment (vhart) [inline PhotoBoothTests/ReviewCellViewModelSpec.swift]: don't commit commented out code

**Pull request merged**